### ISSUE TITLE
Chinese Translation Issue Summary

### ISSUE THREAD

**Issue by JerryHan3** [AlexModGuy/AlexsCaves#480]
TLDR: Current translation has many problem, and lacks respect to previous translators. We hope #479 can be merged to revert back to better translation. 
----
Here's a summary of the Chinese translation issues recently in PR #433 , #436 , #442 and #479. 
Before all these stuffs happened, @Don-Trueno had made a Chinese translation, which is accepted in #433 . This version of translations is a result that involves many mod players and many improper or controversy translations was resolved under discussion between them. Overall, the translation in #433 has a **better quality**. 
Then, @DocOrlande published their version of Chinese translation in #436 . He ignored the existing translation, and directly published his version into this repo WITHOUT discussion with the players' community. Therefore, there're lots of mistakes in this version, such as self-including _Logitech F710_ game console into an item's name, not following existing formats in Chinese translations of vanilla games, and including a Chinese slang in a mob's name that doesn't exist in the original text. This problems means that Doc's version doesn't match the [Chinese Translation Standardization](https://minecraft.fandom.com/zh/wiki/Minecraft_Wiki:ST), might cause misunderstanding in Chinese players community. What's more, rewriting an existing translation is IMPOLITE. His action made all efforts by the community fade away, which lacks respect to those who contribute to the previous version. In conclusion, Doc's version of Chinese translation **should be abandoned**. However, maybe due to Alex's little mistake, this problematic version was still merged into ths main repository in #442 . 
Now @Don-Trueno and I fixed the code conflicts in `zh_cn.json`, made some more developments (including some proper translations by @DocOrlande ), and created a new PR at #479 . Hope Alex can have a clear insight of the current condition, and merge #479 for a better Chinese translations. Many thanks!

**Comment by JerryHan3** [comment]
省流：当前的翻译存在诸多问题，并且不尊重先前的译者。我们希望 PR #479 可以被合并，以恢复到之前更好的翻译。
----
这是一个对现在中文翻译情况，即在 #433 、#436 、 #442 、 #479 中发生的一系列情况的总结。
在这一切发生之前， @Don-Trueno 已经撰写了一份中文翻译，并且在 #433 中被采纳。 这份翻译是众多中文模组玩家的成果，他们针对诸多不当、有争议的译文进行了讨论和更正。总体来讲， #433 中的翻译质量更高。
随后， @DocOrlande 在 #436 中提交了他的翻译版本。他没有注意到已经存在的译文，并且没有和社群讨论就直接发布了自己的译文版本。因此，这一版译文存在诸多问题，比如擅自添加Logitech F710

**Comment by JerryHan3** [comment]
~~Why Github doesn't set a confirmation when closing issues... I accidentally touched that button qwq~~
